src/entrypoints/core.js

```diff
@@ -66,11 +74,23 @@ function main() {
   }
   if (localStorage.tokens) {
     window.tokens = JSON.parse(localStorage.tokens);
-    window.hassConnection = init(null, window.tokens.access_token).catch((err) => {
-      if (err !== ERR_INVALID_AUTH) throw err;
+    if (window.tokens.expires === undefined) {
+      window.tokens.expires = Date.now() - 1;
```

Comment:
Why is this?

Reply:
For migration. The refresh token in local storage from previous version may not have expires field.

Reply:
This is the kind of things that would be good to have a comment added 👍